<html>
<head>
<title>PyException_GetContext</title>
<meta name='robots' content='noindex,nofollow'>
<meta name='generator' content='GLOBAL-5.8.1'>
</head>
<body text='#191970' bgcolor='#f5f5dc' vlink='gray'>
<pre>
<a href='../S/2789.html#L254'>PyException_GetContext</a>  254 Objects/exceptions.c     PyObject *res = PyException_GetContext(self);
<a href='../S/3106.html#L96'>PyException_GetContext</a>   96 Python/errors.c             while ((context = PyException_GetContext(o))) {
<a href='../S/3138.html#L1626'>PyException_GetContext</a> 1626 Python/pythonrun.c             context = PyException_GetContext(value);
</pre>
</body>
</html>
